### PR TITLE
switched from Uint64List to Uint32List to be usable in the browser

### PR DESCRIPTION
**Pull request by denniskaselow**
had to change some tests, best double check them

And I am not 100% certain about the counters, as I'm not using them, but the 63 in there only seems to relate to the max value of int, so I didn't touch those.

### CODE REVIEW

File: lib/src/bit_array.dart [hunk 1]
```diff
@@ -223,6 +223,30 @@ class BitArray implements BitSet {
...
+          return false;
+        }
+      }
+      return true;
+    }
+    return false;
+  }
+
+  @override
+  int get hashCode =>
```
Comment (isoos) [inline lib/src/bit_array.dart]: I'm wondering if both `hashCode` and `==` could be done with one the iterable methods that return either the ints or the 32bit ints. That way they could go inside BitSet. WDYT?

Comment (denniskaselow) [reply lib/src/bit_array.dart]: I'd just need to change every class from `implements BitSet` to `extends BitSet`. If that's okay, I can change it.

Comment (isoos) [reply lib/src/bit_array.dart]: Sure, please do so...

**Comment by denniskaselow** [comment]
You probably should add some tests for `and` and `or` of `CompositeSet` because they didn't do anything. I fixed the obvious bug but it will still fail if the `CompositeSet` was created with a `BitSetChunk` which contains a `BitSet` that isn't a `BitArray`. Because https://github.com/isoos/bit_array/blob/master/lib/src/composite_set.dart#L123 uses `asBitArray` and https://github.com/isoos/bit_array/blob/master/lib/src/composite_set.dart#L21 returns a new `BitArray` which will not change anything in `a._set`.

**Comment by isoos** [comment]
Good catch, thanks! I'll take a fresh second look tomorrow and merge this.

**Comment by isoos** [comment]
FYI: published 2.0

**Pull request merged**